[WIP] Create NEWS file (#59)
EpiForeSITE/epitraxr: NEWS.md

# epitraxr 0.3.2

* Create grouped reports

> * Create SLCounty-style report  

Reviewer: ```suggestion
* Enable PDF output for some reports 
```

* Add threshold parameter to `get_trend()`
* Add new report combining monthly and YTD stats
* Complete R package checklist

# epitraxr 0.2.2

> * Verify piped script matches original script behavior

Reviewer: ```suggestion
* Minor fixes to verify piped script matches original script behavior
```

# epitraxr 0.2.1

* Add method for setting config programmatically

# epitraxr 0.2.0

> * Support pipe operator and add report generation functions for EpiTrax object

Reviewer: ```suggestion
* Introduce a new EpiTrax object
* Support pipe operator
* Add report generation functions for EpiTrax object
```
